automation/capitanodoc/markdown.ts

```diff
@@ -24,7 +24,7 @@ import { CapitanoCommand, Category, Document, OclifCommand } from './doc-types';
 import * as utils from './utils';
 
 function renderCapitanoCommand(command: CapitanoCommand): string[] {
-	const result = [`## ${ent.encode(command.signature)}`, command.help];
+	const result = [`## ${ent.encode(command.signature)}`, command.help!];
```

Comment:
The `*` command doesn't have any help so I had to update the types to allow it to be undefined and then `!` it here as it won't actually be undefined here